good-enough-practices-for-scientific-computing.tex

```diff
@@ -914,7 +914,7 @@ \subsection*{Tool-Based Versioning}
 
   A version control system stores snapshots of a project's files in a
   repository. Users can modify their working copy of the project at
-  will, and then commit changes to the repository when they wish to
+  will, and then 'commit' changes to the repository when they wish to
```

Comment:
This part is intended for people that do not know what tool-based version control is. For them, commit is a technical term. We could also explain it. E.g.

>Users can modify their working copy of the project at will, and then add their changes to the repository by making a so-called 'commit', when they wish to ...

Other suggestions?